solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/ListViewController.swift

```diff
@@ -7,6 +7,10 @@
 
 import UIKit
 
+protocol ListViewControllerProtocol: AnyObject {
```

Comment:
No caso esse delegate seria uma dependencia da própria view (migre a declaração do protocolo para a classe da view), pois todos quem dependem dessa view podem observar a ação de seleção de um repository, assim a gente deixa de acoplar view da view controller... E a ação desse delegate poderia ser algo como didSelectRepository(at:_)
```suggestion
protocol ListViewDelegate: AnyObject {
```